pyvrp/cpp/Individual.cpp

```diff
@@ -262,7 +373,11 @@ std::ostream &operator<<(std::ostream &out, Individual const &indiv)
     auto const &routes = indiv.getRoutes();
 
     for (size_t idx = 0; idx != routes.size(); ++idx)
-        out << "Route #" << idx + 1 << ": " << routes[idx] << '\n';
+    {
+        out << "Route #" << idx + 1;
+        out << " (type " << routes[idx].vehicleType() << ")";
+        out << ": " << routes[idx] << '\n';
```

Comment:
We can also only print vehicle types if we have more than one, but since we don't have access to problemdata here it means we should check that first or only print the type if it is not 0? Any preference?